forms-html/rand_hewr/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+
+</head>
+
+<body>
+    <form>
+        <label for="fname">Enter your Email:</label><br>
+        <input type="text" id="fname" name="fname" required><br>
+        <label for="lname">Usser Name:</label><br>
+        <input type="text" id="lname" name="lname">
+        <br>
+        <label for="lname">Password:</label>
+        <br>
+        <input type="text" id="lname" name="lname"><br>
+        
+        <label for="quantity">Quantity (between 1 and 5):</label><br>
+<input type="number" id="quantity" name="quantity" min="1" max="5"><br>
+<label for="points">Points (between 0 and 50):</label>
+<input type="range" id="points" name="Volume" min="0" max="10">
+</form>
+<p> radio </p>
+<form action="">
+    <input type="radio" name="gender" value="male"> Male<br>
+    <input type="radio" name="gender" value="female"> Female<br>
+    <input type="radio" name="gender" value="other"> Other<br>
+
+  
+   
+
+
+
+
+     
+
+
+        <p>checkbox</p>        
+        <input type="checkbox" id="bike" name="bike" value="bike">
+        <label for="bike">I have a bike</label>
+        <br>
+        <input type="checkbox" id="car" name="car" value="car">
+        <label for="car">I have a car</label>
+        <br>
+        <input type="checkbox" id="boat" name="boat" value="boat">
+        <label for="boat">I have a car</label>
+        <br>
+
+        <p>button</p>
+        <input type="button" value="click me!">
+        <br>
+        <label for="select-color" >Select your favorite color:</label>
+        <input id="select-color" type="color" name="color" value="#000000">
+        <br>
+        <label for="date-before-80">Enter a date before 1980-01-01:</label>
+        <br>
+        <input type="date" id="date-before-80" max="1998-01-01" value="12-11-1979" >
+        <br>
+        <p>Date and time</p>
+        <br>
+        <label for="date-of-birth">Birthday(Date and time):</label>
+        <input type="datetime-local" id="date-of-birth" >   
+
+
+
+
+
+
+
+
+
+
+    <p>submit botton </p>
+
+    <button type="button">submit </button>
+<p>reset form button </p>
+<button type="button">reset  </button>
```

Comment:
for the reset, we use type="reset".
https://www.w3schools.com/tags/att_input_type_reset.asp